### ISSUE TITLE
Website: Pagination/filter not working when categories is set to false?

### ISSUE THREAD

**Issue by giabaio** [quarto-dev/quarto-cli#2466]
### Bug description

Hi all,
I can provide more detailed reprex but I wonder whether this is something that has been explored and has a simple solution? In a nutshell, in a listing page, if I set `categories` to `false`, the pagination and filter box still appear on the page, but they are not clickable. The first page of the many that get created (because I set `page-size: 7`) are simply not shown.

When I set `categories` to `true`, then things work as expected; I can use the filter by clicking on the box and typing whatever I want and I can click on the numbers to navigate to the other group of items.

Is this an obvious bug?
Thanks

### Checklist

- [ ] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [ ] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [ ] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by dragonstyle** [comment]
It doesn't seem quite as simple as just turning off categories. The following listing page (for me) supports sorting and filtering without issue:

```
---
title: Test Listing
listing: 
  - id: post
    contents: "blog/**/*.qmd"
    feed: true
    categories: false
    sort:
      - "date desc"
    max-description-length: 250
    filter-ui: true
    sort-ui: true
    page-size: 3
page-layout: full
title-block-banner: true
---

::: {#post}
:::
```

I tried various types of listings. If you have an example with source I could look at that would help immensely.

**Comment by giabaio** [comment]
Ha --- interesting.... Let me try and prepare a reprex!

**Comment by dragonstyle** [comment]
That would be amazing - thank you.

**Comment by giabaio** [comment]
Thanks again for this. I've put enough of the actual site to replicate the issue [here](https://github.com/giabaio/reprex-site). I've set up `./talks/index.qmd` to specify `categories: false` and at least on my rendering, the pagination and filter are broken. But they work if I set `categories: true`. 

I do have some relatively minor `css` customisation (under `./css`), but I don't think it should impact on `page-layout: full`, which is what `./talks/index.qmd` is set at.

Also, I've updated my quarto version (I couldn't install the very latest commit from `quarto-cli` (it would not work --- I see it's just been pushed, so maybe something not quite right? I'm on Linux/Ubuntu 20.04), but `b48b422` does, so I'm on that.

Hope this is enough to check I'm not missing something obvious -- but please shout if so!
Thanks

**Comment by dragonstyle** [comment]
Thank you I was able to reproduce the issue immediately - really appreciate the time you put into making it easy of me. A fix should be available shortly!

**Comment by giabaio** [comment]
Thank *you*!

**Comment by giabaio** [comment]
Just following up on this --- would the issue be due to some default `z-index` value that is too low and so the rest of the page sits on top of it?...

**Comment by dragonstyle** [comment]
Ahh sorry for losing track of this - this is fixed in the current pre-release builds (here is the fix, z-index related as you suspected). 

8c475aa9a88550e6bd1c27a7abea3adf313be81e

**Comment by giabaio** [comment]
Thank you! This has solved the problem I was originally facing. It seems however, that it may have broken the whole thing in case a page was on full layout *and* had categories on (eg `blog` or `research` in the [reprex](https://github.com/giabaio/reprex-site) --- I've slightly changed/added some files, but it should be showing this behaviour anyway...).

I guess same problem with the sidebar (given by the categories list) being behind some other object?
Sorry to be a pain!

**Comment by dragonstyle** [comment]
Thanks for your continued testing - I was able to see exactly that issue / regression as well and get a fix made. 

This fix is in [v1.2.147](https://quarto.org/docs/download/prerelease.html)

**Comment by giabaio** [comment]
Thank you @dragonstyle! This is very helpful --- works perfectly now!
